scripts/qa_crowbarsetup.sh

```diff
@@ -5302,6 +5321,10 @@ function onadmin_crowbar_nodeupgrade
         fi
 
         if [[ $want_nodesupgrade ]]; then
+            local upgrade_mode=`safely crowbarctl upgrade status --format plain --filter=suggested_upgrade_mode`
+            if [[ "$upgrade_mode" == "normal" ]]; then
```

Comment:
was expecting to re-use the upgrade_mode var for adding extra steps in the future for the non-disruptive part.